packages/good-design/src/apps/goodid/idcard/GoodIdCard.tsx

```diff
@@ -48,10 +47,38 @@ const CardRowItem = withTheme({ name: "CardRowItem" })(
 );
 
 const GoodIdCard = withTheme({ name: "GoodIdCard", skipProps: "credentialsList" })(
-  ({ credentialsList, account, isWhitelisted, avatar, fullname, expiryDate, ...props }: GoodIdCardProps) => {
+  ({ account, isWhitelisted, avatar, fullname, expiryDate, ...props }: GoodIdCardProps) => {
     const { title, subHeading, subContent, footer } = props.fontStyles ?? {};
     const truncatedAccount = truncateMiddle(account, 11);
 
+    const certificates = useAggregatedCertificates(account);
+
+    const verifiedValues = useMemo(() => {
+      let Age: string | null = null,
+        Gender: string | null = null,
+        Country: string | null = null;
+
+      const filtered = certificates?.filter(
+        certificate => certificate.typeName === "Identity" || certificate.typeName === "Location"
+      );
```

Comment:
```suggestion
      const filtered = certificates?.filter(({ typeName }) => ["Identity", "Location"].includes(typeName))
```

Reply:
and you do not need it. you have all 4 types, you just put type from item to the CardRowItem and display corresponding data.

Reply:
@L03TJ3 one of certificates is the same for 3 types, but in result collection it will have 3 items having different type

so you display it 3 times but with passing different type to the row item component which will display different label & data

Reply:
@johnsmith-gooddollar I don't need them all 4.
What we need is the values of the 'Age, Gender, Location' certificates.
so in any way there is a filter needed. Either I filter out the identity certificate and use the remaining three. Or it's like I have done and use the identity certificate to get the gender and location

Reply:
@L03TJ3 you just go over all certificates from useAggregated, you skip Identity Type and you display the value which corresponds the type. Please fix it

Reply:
if it doesn't clear we could discuss tomorrow at sync